js_modules/dagit/packages/core/src/graph/SVGViewport.tsx

```diff
@@ -119,21 +124,31 @@ const PanAndZoomInteractor: SVGViewportInteractor = {
             viewport.adjustZoomRelativeToScreenPoint(scale, {x, y});
           }}
         />
-        <Icon
-          iconSize={17}
-          icon="zoom-out"
-          style={{color: Colors.LIGHT_GRAY1, marginTop: 12}}
-          onClick={() => {
-            const x = viewport.element.current!.clientWidth / 2;
-            const y = viewport.element.current!.clientHeight / 2;
-            viewport.adjustZoomRelativeToScreenPoint(MIN_ZOOM, {x, y});
-          }}
-        />
+        <Box margin={{top: 8}}>
+          <IconButton
```

Comment:
It seems a bit silly to put a -4px left on IconButton but not put a top margin on it at the same time. Maybe this could be left: -4 here, or both could go into the CSS below.

Reply:
The positioning tweak is strictly to get the icon to line up with the slider.